guides/source/getting_started.md

```diff
@@ -194,6 +165,38 @@ of the files and folders that Rails creates by default:
 |.gitignore|This file tells git which files (or patterns) it should ignore. See [GitHub - Ignoring files](https://help.github.com/articles/ignoring-files) for more information about ignoring files.|
 |.rubocop.yml|This file contains the configuration for RuboCop.|
 |.ruby-version|This file contains the default Ruby version.|
+|.devcontainer/|This folder contains the Dev Container configuration|
+
+### Opening the Blog Application in a Dev Container
+
+A Dev Container is a fully functional development environment running in a container
```

Comment:
I suggested explaining what a Dev Container is at the start of the guide when we mention Dev Containers for the first time, so maybe if we decide to go that route, we can remove this introduction of what a Dev Container is?